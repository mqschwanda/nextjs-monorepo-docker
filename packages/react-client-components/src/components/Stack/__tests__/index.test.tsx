import { render } from '@testing-library/react';
import { Stack } from '..';

describe('@mqs/react-client-components', () => {
  describe('components', () => {
    describe('<Stack />', () => {
      it('renders', () => {
        const testId = 'Stack';

        const { getByTestId, asFragment } = render((
          <Stack
            testId={testId}
          >
            <div>
              one
            </div>
            <div>
              two
            </div>
          </Stack>
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
