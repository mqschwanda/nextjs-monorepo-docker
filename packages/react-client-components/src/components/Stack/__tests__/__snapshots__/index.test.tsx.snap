// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-client-components components <Stack /> renders 1`] = `
<DocumentFragment>
  <div
    class="MuiStack-root css-nen11g-MuiStack-root"
    data-testid="Stack"
  >
    <div>
      one
    </div>
    <div>
      two
    </div>
  </div>
</DocumentFragment>
`;
